core/BUILD.bazel

```diff
@@ -4,33 +4,16 @@ java_library(
     exports = ["//api"],
 )
 
-java_library(
-    name = "inprocess",
-    srcs = glob([
-        "src/main/java/io/grpc/inprocess/*.java",
-    ]),
-    visibility = ["//visibility:public"],
-    deps = [
-        ":internal",
-        "//api",
-        "//context",
-        "@com_google_code_findbugs_jsr305//jar",
-        "@com_google_errorprone_error_prone_annotations//jar",
-        "@com_google_guava_guava//jar",
-        "@com_google_j2objc_j2objc_annotations//jar",
-    ],
-)
-
 java_library(
     name = "internal",
+    visibility = ["//visibility:public"],
```

Comment:
internal should not be public. Why did you need this? It was already allowing subpackages.

Reply:
done